.github/workflows/release.yml

```diff
@@ -8,30 +8,31 @@ on:
         description: Bump version number
         options: [major, minor, patch]
         default: patch
-  pull_request:
-    types: [closed]
+  pull_request: { types: [closed] }
 jobs:
   bump:
     if: github.event_name == 'workflow_dispatch'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - run: |
-          git config --global user.name Olivaw[bot]
-          git config --global user.email 64868532+iterative-olivaw@users.noreply.github.com
+      - uses: actions/checkout@v3
+      - uses: iterative/setup-cml@v1
+      - name: Bump version
+        run: |
+          cml ci
           git checkout -b bump/$(npm version ${{ github.event.inputs.bump }})
+      - name: Create PR
+        run: |
           git push --set-upstream origin --follow-tags HEAD
-          gh pr create --title "Bump version to $(git describe --tags)" --body "Approve me 🤖"
-          gh pr merge --squash --auto
+          cml pr create --title "Bump version to $(git describe --tags)" --body "Approve me 🤖" --squash
         env:
           GITHUB_TOKEN: ${{ secrets.ADMIN_GITHUB_TOKEN }}
   release:
     if: github.event.pull_request.merged && startsWith(github.head_ref, 'bump/')
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - run:
-          gh release create --generate-notes {--title=CML\ ,}$(basename ${{
-          github.head_ref }})
+          gh release create --draft --generate-notes {--title=CML\ ,}$(basename
```

Comment:
<sup>[justification needed]</sup>

Reply:
human notes currently are far superior to the auto-generated ones

Reply:
But you can edit them, regardless of the `--draft` option 🤔

Reply:
subscribers will get notifications pre-edit not post-edit. This is v. bad.

Reply:
Good point